algorithms/kernel/k_nearest_neighbors/oneapi/bf_knn_classification_model_ucapi_impl.h

```diff
@@ -0,0 +1,179 @@
+/* file: bf_knn_classification_model_ucapi_impl.h */
+/*******************************************************************************
+* Copyright 2014-2019 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+/*
+//++
+//  Implementation of the class defining the K-Nearest Neighbors (kNN) model
+//--
+*/
+
+#ifndef __BF_KNN_CLASSIFICATION_MODEL_UCAPI_IMPL_H__
+#define __BF_KNN_CLASSIFICATION_MODEL_UCAPI_IMPL_H__
+
+#include "algorithms/k_nearest_neighbors/bf_knn_classification_model.h"
+#include "data_management/data/numeric_table_sycl_homogen.h"
+
+#include "daal_defines.h"
+
+using namespace daal::services;
+using namespace daal::oneapi::internal;
+
+namespace daal
+{
+namespace algorithms
+{
+namespace bf_knn_classification
+{
+namespace interface1
```

Comment:
Is this file contains definitions of the internal entities? If yes, it should be not in `interface` namespace but in `internal`